pimd/pim_register.c

```diff
@@ -346,7 +346,7 @@ pim_register_recv (struct interface *ifp,
 	  zlog_debug("%s: Sending register-Stop to %s and dropping mr. packet",
 	    __func__, "Sender");
 	/* Drop Packet Silently */
-	return 1;
+	return 0;
```

Comment:
I'm not quite following what  a return of 0/1 signifies here. If 0 means failing to do what we are supposed to do, why are we returning 1 when we fail to find an upstream router few lines below and still returning 1 at line 365?  I'm not saying it is wrong, but please confirm that these cases are all correct. 

Something unrelated to this commit, but while we are at this spot a couple of lines above, should "Sender" be replaced with the value of src_addr?